release-calendar/src/client/components/Calendar.tsx

```diff
@@ -38,6 +38,10 @@ export class Calendar extends React.Component<CalendarProps, {}> {
           }}
           plugins={[dayGridPlugin, timeGridPlugin]}
           eventSources={this.props.events}
+          contentHeight={480} //will be 430 when header is added
```

Comment:
Can you make this a top-level const in this file? (e.g., `const CALENDAR_CONTENT_HEIGHT = 480;`)
Generally speaking if a value is not one of: `0`, `1`, `2`, `true`, `false`, `null`, `undefined`, or `''`, consider making it a const. If you end up reusing this value for a different component it makes it easier to change all the values at one :)

Reply:
oh, sure!